packages/winston-log-enricher/tests/versioned/winston.tap.js

```diff
@@ -24,7 +25,15 @@ tap.test('Winston instrumentation', (t) => {
 
   t.beforeEach(() => {
     helper = utils.TestAgent.makeInstrumented()
-    helper.agent.config.application_logging = { enabled: true, metrics: { enabled: true } }
+    helper.agent.config.application_logging = {
+      enabled: true,
+      metrics: {
+        enabled: true
+      },
+      forwarding: {
```

Comment:
we should also probably add tests where it does not add logs to aggregator through the various combinations of config differences.  we are implicitly asserting it is only adding logs in transaction context because of the test you added but we're missing this one scenario

Reply:
Okay, tests added.